package.json

```diff
@@ -43,8 +43,8 @@
         "cov-report-html": "nyc report --reporter=html",
         "start": "ts-node ./scripts/credentials.ts -- . && webpack-dev-server -d --config @here/harp-examples/webpack.config.js",
         "build": "npm run build-bundle && npm run build-examples",
-        "build-examples": "ts-node ./scripts/credentials.ts -- dist/examples && webpack -d --config @here/harp-examples/webpack.config.js",
-        "build-bundle": "cd @here/harp.gl && webpack",
+        "build-examples": "ts-node ./scripts/credentials.ts -- dist/examples && webpack -p --config @here/harp-examples/webpack.config.js",
+        "build-bundle": "cd @here/harp.gl && webpack && NODE_ENV=production webpack",
```

Comment:
`&& webpack && NODE_ENV=production webpack` I guess here webpack run twice once in dev mode and second in production.

Reply:
Yes, this creates minified bundles for production(*.min.js) and unminified bundles(*.js).

Reply:
@MPursche Look at `package.json` in `@here/harp.gl` - we already build two bundles in `prepare` step used by `npm pack`.
So, this  change `build-bundle` is not necessary.